### PR TITLE
Initializing react

### PR DESCRIPTION
**Pull request by jlrodney**

### CODE REVIEW

File: Procfile.dev [hunk 1]
```diff
@@ -0,0 +1,2 @@
+web: cd recipebookbook-frontend && PORT=3000 npm start
```
Comment (jonurry) [inline Procfile.dev]: I think it's more conventional to name the frontend folder as 'client'

File: recipebookbook-frontend/public/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (jonurry) [inline recipebookbook-frontend/public/index.html]: we should change the default title

File: recipebookbook-frontend/public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```
Comment (jonurry) [inline recipebookbook-frontend/public/manifest.json]: we should change the defaults to reflect our actual app

File: recipebookbook-frontend/src/components/PostsContainer.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  }
+
+
+  render() {
+    return (
+      <div className="Posts-container">
+
+        <NewPostForm onNewPost={this.addNewPost}/>
+        {this.state.posts.map( post => {
+          if (this.state.editingPost === post.id) {
```
Comment (JonathanAndrews) [inline recipebookbook-frontend/src/components/PostsContainer.js]: if (this.state.editingPostId === post.id) {

Not certain but I think this is what the line should be
@m-rcd @jonurry

**Pull request merged**